feature-libs/product-configurator/rulebased/components/variant-carousel/configurator-variant-carousel.component.spec.ts

```diff
@@ -0,0 +1,192 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ConfiguratorVariantCarouselComponent } from './configurator-variant-carousel.component';
+import { Observable, of } from 'rxjs';
+import { Product, ProductService, TranslationService } from '@spartacus/core';
+import {
+  CommonConfigurator,
+  ConfiguratorModelUtils,
+  ConfiguratorRouter,
+  ConfiguratorRouterExtractorService,
+  ConfiguratorType,
+} from '@spartacus/product-configurator/common';
+import {
+  Configurator,
+  ConfiguratorCommonsService,
+} from '@spartacus/product-configurator/rulebased';
+import { ConfiguratorTestUtils } from '../../testing/configurator-test-utils';
+import { Component, Input, Pipe, PipeTransform } from '@angular/core';
+import { CommonConfiguratorTestUtilsService } from '../../../common/testing/common-configurator-test-utils.service';
+
+const PRODUCT_DESCRIPTION = 'Here is a product description';
```

Comment:
Let's have the formatting of constants unified. Some are upper case, others not. I would rather do them in camel case style

Reply:
Done!